ISSUES.md

```diff
@@ -106,6 +108,7 @@
 |NODE_SKIN_WITH_NON_SKINNED_MESH|Node has skin defined, but mesh has no joints data.|Error|
 |NODE_WEIGHTS_INVALID|The length of weights array (`%1`) does not match the number of morph targets (`%2`).|Error|
 |SCENE_NON_ROOT_NODE|Node `%1` is not a root node.|Error|
+|SKIN_IBM_ACCESSOR_WITH_BYTESTRIDE|bufferView.byteStride must not be defined for buffer views used by IBM accessors.|Error|
```

Comment:
In the long description here, can we spell out IBM to `inverseBindMatrices`?

Reply:
Since the error is already attributed to `inverseBindMatrices` property, maybe spell out IBM in the description using separate words?

Reply:
Either way is fine.  Reading the `ISSUES.md` list by itself didn't make this clear, as there was no attribution.  But I think it's fine for the reported error to include the name `inverseBindMatrices` even when it is attributed.